### DISCUSSION TITLE
关于指定位置换行的讨论

### DISCUSSION THREAD

**Discussion by hooray** [zhensherlock/watermark-js-plus#855] [Q&A] [answered]
需求是这样的，水印内容是后端返回的，而水印文本是通过 input textarea 配置的，textarea 是可以进行换行的，相当于文本内容里是带有`\n`隐式换行符的。

而插件的多行文本是通过设置宽高和字号进行换行的，除了改成富文本的形式（后端也返回富文本），似乎没办法满足`\n`换行符的场景。

另外富文本也发现一个问题，就是不能处理`<br>`，例如`<div>123<br>321</div>`，水印无法正常显示。

**Comment by zhensherlock** [comment]
感谢反馈，近期会发布多行文本支持`\n`换行和富文本支持`<br>`的功能

**Comment by zhensherlock** [comment, marked as answer]
你好，请更新至最新版本v1.5.4，上述两个功能均已支持。
